extensions/2.0/Khronos/KHR_audio/README.md

```diff
@@ -0,0 +1,314 @@
+      }
+    }
+  ]
+}
+```
+
+## glTF Schema Updates
+
+This extension consists of three primary data structures: Audio Data, Audio Sources, and Audio Emitters. Data, sources and emitters are defined on an `KHR_audio` object added to the `extensions` object on the document root.
+
+The extension must be added to the file's `extensionsUsed` array and because it is optional, it does not need to be added to the `extensionsRequired` array.
+
+#### Example:
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    }
+    "extensionsUsed" : [
+        "KHR_audio"
+    ],
+    "scenes": [...],
+    "nodes": [...],
+    "extensions": {
+        "KHR_audio": {
+            "audio": [...],
+            "sources": [...],
+            "emitters": [...]
+        }
+    }
+}
+```
+
+### Audio Data
+
+Audio data objects define where audio data is located. Data is either accessed via a bufferView or uri.
+
+When storing audio data in a buffer view, the `mimeType` field must be specified. Currently the only supported mime type is `audio/mpeg` for use with MP3 files. MP3 was chosen due to its wide support across browsers and 3D engines as well as its lossy compression with variable bitrate. Other supported audio formats may be added via another extension.
+
+Note that in tools that process glTF files, but do not implement the KHR_audio extension, external files referenced via the `uri` field may not be properly copied to their final destination or baked into the final binary glTF file. In these cases, using the `bufferView` property may be a better choice assuming the referenced `bufferView` index is not changed by the tool. The `uri` field might be a better choice when you want to be able to quickly change the referenced audio asset.
+
+#### `bufferView`
+
+The index of the bufferView that contains the audio data. Use this instead of the audio source's uri property.
+
+#### `mimeType`
+
+The audio's MIME type. Required if `bufferView` is defined. Unless specified by another extension, the only supported mimeType is `audio/mpeg`.
+
+#### `uri`
+
+The uri of the audio file. Relative paths are relative to the .gltf file.
+
+### Audio Sources
+
+Audio sources define the playing state for a given audio data. They connect one audio data to zero to many audio emitters.
+
+#### `gain`
+
+Unitless multiplier against original audio file volume for determining audio source loudness.
+
+#### `loop`
+
+Whether or not to loop the specified audio when finished.
+
+#### `autoPlay`
+
+Whether or not to play the specified audio when the glTF is loaded.
+
+#### `audio`
+
+The index of the audio data assigned to this clip.
+
+### Audio Emitter
+
+Positional or global sinks for playing back audio sources.
+
+#### `type`
+
+Specifies the audio emitter type.
+
+- `positional` Positional audio emitters. Using sound cones, the orientation is `+Z` having the same front side for a [glTF asset](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#coordinate-system-and-units). 
```

Comment:
This is inconsistent with KHR_lights_punctual which defines spotlights as facing -Z.

Reply:
Agreed, -Z forward makes the most sense. It's also in line with how the WebAudio spec defines audio listeners. I can't find the details on panner node, besides it seemingly defaulting to +X forward? This will break existing files but better to do it now.

Reply:
todo: @antpb change to negative Z